objects/git_repo.rb

```diff
@@ -100,9 +100,9 @@ def pull
         "cd #{@path}",
         "master=#{Shellwords.escape(@master)}",
         'git config --local core.autocrlf false',
-        'git fetch --force --all --quiet',
-        'git reset --hard --quiet origin/${master}',
+        'git reset origin/${master} --hard --quiet',
         'git clean --force -d',
+        'git fetch --quiet',
```

Comment:
@mbao01 why did you remove `--force --all`? I believe, they are here for a reason.

Reply:
@yegor256 It seems like a simple change. I would not touch this if I did not encounter an issue with it while working on this integration. Maybe a good thing will be to leave it as is and just add a @todo marker. Please let me know how to proceed.